### ISSUE TITLE
'Could not find definition' in nested property definition

### ISSUE THREAD

**Issue by dieseldjango** [rjsf-team/react-jsonschema-form#988]
### Prerequisites

- [ x] I have read the [documentation](https://github.com/mozilla-services/react-jsonschema-form/blob/master/README.md#readme);
- [x ] In the case of a bug report, I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.

### Description
I'm working with a schema that was provided to me in multiple files, which I bundled using json-schema-ref-parser. I believe the schema is valid, as I can validate against it with AJV. However, when I pass this schema into the Form component, it fails with this error:

```
Error: Could not find a definition for #/properties/peripherals/properties/printerType/definitions/peripherals_PrinterProperties.
    at findSchemaDefinition (utils.js:484)
    at computeDefaults (utils.js:187)
    at utils.js:205
    at Array.reduce (<anonymous>)
    at computeDefaults (utils.js:202)
    at utils.js:205
    at Array.reduce (<anonymous>)
    at computeDefaults (utils.js:202)
    at utils.js:205
    at Array.reduce (<anonymous>)
```
### Steps to Reproduce
1. Render a form passing in
[schema.zip](https://github.com/mozilla-services/react-jsonschema-form/files/2229788/schema.zip).

See https://jsfiddle.net/dieseldjango/n5u2wwjg/108639/
#### Expected behavior
Form to render.

#### Actual behavior
Aforementioned error.

### Version
1.0.3

**Comment by epicfaace** [comment]
I believe the issue is that you can only use $ref to reference definitions that are in the `definitions` property in the schema (i.e., all referenced paths should start with `#/definitions/...`).

**Comment by glasserc** [comment]
Refs #954.

**Comment by newt10** [comment]
Seems like this should be fixed by [#1506 ](https://github.com/rjsf-team/react-jsonschema-form/pull/1506).
@dieseldjango if this issues still persists, please re-open or file a new one with the details.
